actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -138,13 +139,15 @@ def _filtered_referrer # :nodoc:
       #   link_to "Profiles", controller: "profiles"
       #   # => <a href="/profiles">Profiles</a>
       #
-      # You can use a block as well if your link target is hard to fit into the name parameter. ERB example:
+      # You can use a block as well to wrap your link around a block of HTML. ERB example:
       #
       #   <%= link_to(@profile) do %>
-      #     <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
+      #     <h4><%= @profile.name %></h4>
```

Comment:
Why do you think using block elements improves the example? This is only valid with HTML5 and I think inline elements still make up the majority of use cases.